src/pages/programs/index.tsx

```diff
@@ -0,0 +1,170 @@
+import * as React from "react";
+import { useState } from "react";
+
+import ArticleSpotlightCard from "../../components/grid-aware/ArticleSpotlightCard";
+import Modal from "../../components/grid-aware/Modal";
+import ProgramBlock from "../../components/grid-aware/ProgramBlock";
+import ProgramsSubtitleBlock from "../../components/grid-aware/ProgramsSubtitleBlock";
+import Spacer from "../../components/grid-aware/Spacer";
+import TextHeader from "../../components/grid-aware/TextHeader";
+import Layout from "../../components/layout";
+import PartnershipSignupForm from "../../components/thirdparty/mailchimp/PartnershipSignupForm";
+import communityDevelopmentImg from "./community-development.jpg";
+import articleSpotlightImage from "./laura-barerra-vera.png";
+import serviceGuideImg from "./sf-service-guide.jpg";
+import shelterConnect from "./shelter-connect.jpg";
+
+export default () => {
+  const [partnershipFormIsOpen, setPartnershipFormIsOpen] = useState(false);
+
+  return (
+    <Layout>
+      <Modal
+        isOpen={partnershipFormIsOpen}
+        setIsOpen={setPartnershipFormIsOpen}
+        contentLabel="Partnership"
+      >
+        <PartnershipSignupForm />
+      </Modal>
+      <TextHeader
+        title="Our Programs"
+        description="ShelterTech's programs are designed to support the underserved community experiencing homelessness and housing insecurity through WiFi connectivity and access to online resources."
+      />
+      <ProgramsSubtitleBlock title="ShelterTech offers the following programs:" />
+      <ProgramBlock
+        title="SF Service Guide"
+        subtitle="an online directory of human services in San Francisco"
+        description={
+          <>
+            <p>
+              The SF Service Guide is an online directory of human services in
+              San Francisco that launched in November 2019 and currently serves
+              1,500 unique monthly users. The directory&apos;s focus is on
+              homelessness and housing services but also covers a variety of
+              other services, from education and legal aid to senior services
+              and re-entry programs. Our goal is to help anyone with access to a
+              smartphone, tablet, or computer find the services they need.
+            </p>
+            <p>
+              SF Service Guide is supported by a grant from the SF Mayor&apos;s
+              Office of Housing and Community Development and verified for
+              accuracy by our data partner, the Homeless Advocacy Project of the
+              JDC.
+            </p>
+          </>
+        }
+        image={{
+          url: serviceGuideImg,
+          alt: "SF Service Guide",
+        }}
+        ctaButtons={[
+          {
+            text: "Visit Site",
+            externalLink: "https://sfserviceguide.org/",
+          },
+          {
+            text: "Donate",
+            internalLink: "https://www.sheltertech.org/donate/",
```

Comment:
Can you leave off the `https://www.sheltertech.org` from all these internal links? Having the full domain here makes the link behave strangely in local development environments, since clicking on the link causes it to open the real, production site instead of navigating there in the local environment. This would also affect staging, as well as any version of the site that's not the production one.

There's two more `internalLink`s lower on the page that should be fixed up as well.